### ISSUE TITLE
Recently added media playlist for all servers

### ISSUE THREAD

**Issue by TopperBG** [Casvt/Plex-scripts#219]
A lot of usefull tools are made here and obviously exprtise is level up.
So I share my toughts here ;)
Plex already have search "all around" broadcast networks AND friends' severs (I'm noob in PlexAPI)
Is it posible to search in friends' servers for eg. "New movies" or "Top rated" or "Unwached titles" ? And respectivly respectively form a watch list or collection.
Just thoughts, but when you have 10-20 active friends you can't know what new/top/rated is uploaded somewhere.

**Comment by Casvt** [comment]
Honestly, I don't have a single clue what you mean. Could you be more clear?

**Comment by TopperBG** [comment]
I have 20-30 friends in my Plex network. Some of them are IMHO maniacs - download old high rated movies in DV/HDR+ formats, some of them all new movies, third of them high rated TV shows etc.
It's hard to go around 20-30 servers and check what's new is downloaded in particular server.
I'm just wondering is the PlexAPI is powerfull enough to parse my friends network, lookup servers and extract new uploaded in libraries. Respectively create lists/collections as "new released movies" "new released shows" "top rated in X server" or "fresh downloaded movies". I hope now it's clear :)

**Comment by Casvt** [comment]
For a library (even a library of another server), you can create a smart collection/playlist that only includes media that was added in the last x days. It gets dynamically updated.

But that's a playlist/collection per library. Smart playlists/collections cannot be across multiple libraries. If you just want a "Newly added movies" playlist per server, that's not a problem though.

On the home page, does it not also show a "Recently added in X" section for each library of each server?

**Comment by TopperBG** [comment]
It's time consuming to go trough all servers... and If I don't wrong smart list stays at each server, what means must go (again) to that particular server to see what new happens.
Ok thanks for attention that was just my thoughts

**Comment by Casvt** [comment]
Yeah but it's also not possible to create a playlist that is across multiple servers. Only thing I can do is make a script that lists any media added in the last x days from all servers you have access to.

**Comment by TopperBG** [comment]
> Yeah but it's also not possible to create a playlist that is across multiple servers. Only thing I can do is make a script that lists any media added in the last x days from all servers you have access to.

Hm, that's a good start ? After that could be parsed (Tautulli, Kometa style) and made playlists/collections ?
